be/src/env/env.h

```diff
@@ -29,19 +29,21 @@ class Env {
 public:
     // Governs if/how the file is created.
     //
-    // enum value                      | file exists       | file does not exist
-    // --------------------------------+-------------------+--------------------
-    // CREATE_IF_NON_EXISTING_TRUNCATE | opens + truncates | creates
-    // CREATE_NON_EXISTING             | fails             | creates
-    // OPEN_EXISTING                   | opens             | fails
-    enum CreateMode {
-        CREATE_IF_NON_EXISTING_TRUNCATE,
-        CREATE_NON_EXISTING,
-        OPEN_EXISTING
+    // enum value                   | file exists       | file does not exist
+    // -----------------------------+-------------------+--------------------
+    // CREATE_OR_OPEN_WITH_TRUNCATE | opens + truncates | creates
+    // CREATE_OR_OPEN               | opens             | creates
+    // MUST_CREATE                  | fails             | creates
+    // MUST_EXIST                   | opens             | fails
+    enum OpenMode {
+        CREATE_OR_OPEN_WITH_TRUNCATE,
+        CREATE_OR_OPEN,
+        MUST_CREATE,
+        MUST_EXIST
```

Comment:
`CREATE_OR_OPEN` starts with a verb, and MUST_CREATE is not. we should keep it consistent.